docs/docs/noir/standard_library/meta/fmtstr.md

```diff
@@ -0,0 +1,13 @@
+---
+title: fmtstr
+---
+
+`std::meta::format_string` contains comptime methods on the `fmtstr` type for format strings.
+
+## Methods
+
+### contents
+
+#include_code contents noir_stdlib/src/meta/format_string.nr rust
+
+Returns the format string contents (that is, without the leading and trailing double quotes) as a `Quoted` value.
```

Comment:
Can you rename this function to `quoted_contents()`? I think that makes it clearer it returns a quoted value, and will help steer away users that are not doing comptime things.